src/main/cpp/subsystems/DriveSubsystem.cpp

```diff
@@ -261,3 +243,22 @@ void DriveSubsystem::ResetOdometry(frc::Pose2d pose) {
   poseEstimator.ResetPosition(GetHeading(), GetModulePositions(), pose);
   // ResetEncoders();
 }
+
+std::optional<IGyroContainer*> DriveSubsystem::GetGyro() {
+  for (int i = m_lastGoodGyroIndex; i < m_gyros.size(); i++) {
+    auto* gyro = m_gyros[i];
+    if (gyro->HasError()) {
+      ConsoleLogger::getInstance().logError(
+          "DriveSubsystem", "Gyro %s has had error '%s'.", gyro->GetName(),
+          gyro->GetStatusMessage());
+      m_lastGoodGyroIndex++;
+      continue;
+    }
+    return gyro;
+  }
+
+  ConsoleLogger::getInstance().logError(
+      "DriveSubsystem", "All gyros have failed. HELP US EVAN!!!!1!!1!%s", "");
```

Comment:
Is this the 3rd log in our code that explicitly calls out evan?

Reply:
@WowMuchDoge